src/gxhash/mod.rs

```diff
@@ -276,4 +286,4 @@ mod tests {
         assert_eq!(3325885698, gxhash32(&[0u8; 1000], 0));
         assert_eq!(3805815999, gxhash32(&[42u8; 4242], 42));
     }
-}
\ No newline at end of file
+}
```

Comment:
I'll see if I can enforce a line ending policy. In the meantime, it's best not to change them